api/hooks/customUserHook/index.js

```diff
@@ -21,15 +21,16 @@ module.exports = function userHook() {
 
     findUser: function (id) {
       return new Promise(function (resolve, reject) {
-        User.findOne(id).exec(function (error, user) {
-          if (error) {
-            reject(error);
-          } else if (!user) {
-            reject({ message: 'User does not exist' });
-          } else {
-            return resolve(user);
-          }
-        });
+        User.findOne(id).populate('identities')
```

Comment:
I want to re-examine where this hook is used. It doesn't really add any value over a raw .find() imo. I initially wrote it before sails supported promises on its queries (or else I didn't realize that it did) so this was just intended as a promise-wrapper for a basic query. That shouldn't really be necessary any more. 

Do we actually want/need the User records to populate their identities?